site/layouts/default.hbs

```diff
@@ -21,7 +21,7 @@
 {{#is page.pageType "theme"}} gc-theme{{/is}}
 {{#is page.pageType "news"}} gc-news{{/is}}
 {{#is page.pageType "advance service"}} gc-advnc-srvc{{/is}}
-">
+" typeof="WebPageElement">
```

Comment:
` typeof="WebPageElement"` should be added in the if statement as well, not just in the else.